[thread] Replace 'could' and 'might' (#4349)
cplusplus/draft: source/threads.tex

\pnum
\begin{note}
If the clock is not synchronized with a steady clock, e.g., a CPU time clock,

> it is possible for these timeouts not to provide useful functionality.

Reviewer: ```suggestion
it is possible that these timeouts do not provide useful functionality.
```

\end{codeblock}

\begin{note}
It is possible for line \#1 not to result in concurrency
because the \tcode{async} call uses the default policy,

> which possibly uses \tcode{launch::deferred},
> in which case it is possible for the lambda not to be invoked until the

Reviewer: ```suggestion
which can use \tcode{launch::deferred},
in which case it is possible that the lambda is not invoked until the
```
I think this reads a bit better. Still incredibly awkward, but maybe this is the best we can do.
